seed/static/seed/locales/en_US.json

```diff
@@ -46,11 +46,11 @@
     "Address Line 2 (Tax Lot)": "Address Line 2 (Tax Lot)",
     "Admin": "Admin",
     "An error occurred while processing the file. Please ensure that your file meets the required specifications.": "An error occurred while processing the file. Please ensure that your file meets the required specifications.",
-    "AND": "AND",
     "and": "and",
+    "AND": "AND",
```

Comment:
Not sure why this happened for all capitalization alternatives...

Reply:
Likely these were changes made prior and not imported into SEED. Should be fine that it changed.